pkg/proxy/apis/config/validation/validation_test.go

```diff
@@ -193,7 +189,19 @@ func TestValidateKubeProxyConfiguration(t *testing.T) {
 			},
 			expectedErrs: field.ErrorList{field.Invalid(newPath.Child("logging.format"), "unsupported format", "Unsupported log format")},
 		},
-	} {
+	}
+
+	if runtime.GOOS != "windows" {
```

Comment:
can this be organized with go build tags / separate files instead?

Reply:
Done